generator/src/main/java/com/scottlogic/deg/generator/GenerateTreeCollectionJson.java

```diff
@@ -13,6 +13,7 @@
 import com.scottlogic.deg.generator.inputs.ProfileReader;
 
 import com.scottlogic.deg.generator.inputs.validation.NoopProfileValidator;
+import com.scottlogic.deg.generator.inputs.validation.ProfileValidator;
```

Comment:
Nothing has changed in this file, except for whitespace, does this package need to be imported - or can the change to this file be 'undone'?